src/components/lyne-autocomplete-list/lyne-autocomplete-list.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { LyneAutocompleteList } from './lyne-autocomplete-list';
+import { newSpecPage } from '@stencil/core/testing';
+
+describe('lyne-autocomplete-list', () => {
+  it('renders', async () => {
+    const {
+      root
+    } = await newSpecPage({
+      components: [LyneAutocompleteList],
+      html: '<lyne-autocomplete-list />'
+    });
+
+    expect(root)
+      .toEqualHtml(`
+        <lyne-autocomplete-list>
+          <mock:shadow-root>
+            <div class="autocomplete-list" tabindex="-1">
+              <ul aria-hidden="" class="autocomplete-list__list" role="presentation"></ul>
+              <p aria-atomic="true" aria-live="polite" class="autocomplete-list__accessibility-hint" role="status"></p>
```

Comment:
i don't exactly understand what you mean by "do this as a separate module"... you think of making a module for the <p> in this case with the according aria-attributes?

Reply:
I war refering to the idea of having a separate module e.g. lyne-aria-live-message, which would receive the messages dynamically from other modules instead of having an aria-live region in every module.